score/audit-checklist.md

```diff
@@ -220,6 +220,25 @@ def transfer(self, _to: Address, _value: int, _data: bytes = None):
     self.Transfer(self.msg.sender, _to, _value, _data)
 ```
 
+In addition to sending tokens between the two addresses, you must leave Eventlog even if tokens are minted or burned. For mint and burn, use a specific address as follows:
```

Comment:
```suggestion
There are special cases of token transfer. When `mint` or `burn`, it is highly recommended to trigger a `Transfer` event log with EOA_ZERO address. It is a common practice implementing `mint` as a token transfer from EOA_ZERO, while `burn` as a token transfer to EOA_ZERO. Please note that EOA_ZERO is not defined in `iconservice` so, you should define the variable as shown in the below example. 
Alternatively, you can define your own event log functions for `mint` and `burn`, but using `Transfer` event log would be more preferable because it gives a room for ICON Tracker to detect the IRC2 token's mint and burn events.
```